packages/webpack-plugin/src/mini-css-support.ts

```diff
@@ -46,8 +46,26 @@ export function injectCssModules(
                 }),
                 media: '',
                 sourceMap: null,
+                assets: {},
+                assetsInfo: {},
             });
 
+            try {
+                let syncCheck = false;
+                cssModule.build(
+                    compilation.options,
+                    compilation,
+                    compilation.resolverFactory.get('normal'),
+                    compilation.compiler.inputFileSystem,
+                    () => (syncCheck = true)
+                );
+                if (!syncCheck) {
+                    throw new Error('Expect CssModule build to be sync');
+                }
+            } catch (e) {
+                throw new Error('CssModule build error inside Stylable integration');
```

Comment:
The original error message and stack is being swallowed

Reply:
I know. I don't thing they add benefit for the user. sine we are applying the build manually. when this error happens the issue is known. if you think it's a must replay.

Reply:
It's useful to have those exposed for bug reports.

Reply:
So error message is now exposed.